java/squeek/veganoption/integration/waila/ProviderJutePlant.java

```diff
@@ -34,9 +35,9 @@ public ItemStack getWailaStack(IWailaDataAccessor accessor, IWailaConfigHandler
 		{
 			float growthValue = ((BlockJutePlant) accessor.getBlock()).getGrowthPercent(accessor.getWorld(), accessor.getPosition(), accessor.getBlockState()) * 100.0F;
 			if (growthValue < 100)
-				toolTip.add(String.format("%s : %.0f %%", I18n.format("hud.msg.growth"), growthValue));
+				toolTip.add(LangHelper.translate("waila.growth", (int) growthValue));
```

Comment:
Yes, it does use the standard string formatting through `Locale`.

With `VeganOption.waila.growth=Growth : %.0f %%` and no typecasts on `growthValue`, the resulting tooltip states `Growth : 0.0 %`.